dojo/tools/bundler_audit/parser.py

```diff
@@ -28,17 +28,19 @@ def get_findings(self, filename, test):
         for warning in warnings:
             if not warning.startswith("Name"):
                 continue
-            advisory_cve = None
+            advisory_id = None
             gem_report_fields = warning.split("\n")
             for field in gem_report_fields:
                 if field.startswith("Name"):
                     gem_name = field.replace("Name: ", "")
                 elif field.startswith("Version"):
                     gem_version = field.replace("Version: ", "")
                 elif field.startswith("Advisory"):
-                    advisory_cve = field.replace("Advisory: ", "")
+                    advisory_id = field.replace("Advisory: ", "")
                 elif field.startswith("CVE"):
-                    advisory_cve = field.replace("CVE: ", "")
+                    advisory_id = field.replace("CVE: ", "")
+                elif advisory_id is None and field.startswith("GHSA"):
```

Comment:
Hi @rh0dy:
I guess `advisory_id is None` and then replacing it will result in:
`AttributeError: 'NoneType' object has no attribute 'replace'`
Did you test this?
Furthermore, I don't recommend to replace `("GHSA: ", "")` as this is relevant information.

Reply:
Hey @manuel-sommer, thanks for the review! I might be missing something here, can you check my understanding as I'm not sure how `AttributeError` would get raised & it shouldn't be removing relevant information from `GHSA`:
- I don't think it performs a `.replace` on `advisory_id`, instead it assigns a string value to `advisory_id` 
- If the finding doesn't contain a `CVE` it will use the `GHSA`, but only if `CVE` isn't set (so that CVE is prioritised)
- The `.replace` is for truncating the beginning part of the line so it's left with only the ID part, e.g. `GHSA: GHSA-j6w9-fv6q-3q52` becomes `GHSA-j6w9-fv6q-3q52`

Reply:
aaah, sorry, I guess my review here is wrong. you can mark it as "resolved".